site/app/templates/grading/electronic/GradingPanelHeader.twig

```diff
@@ -56,6 +57,15 @@
             {{ self.renderPanelPositionSelector('student_info') }}
             </span>
         {% endif %}
+        <span class="grade-panel" id="solution_ta_notes_btn">
+            <button title="Show Solution and TA notes"
+                    class="invisible-btn key_to_click"
```

Comment:
you don't need the "key_to_click" class for HTML buttons, the `onclick/click` event will fire on mouse press or mouse enter